src/components/customers/utils.ts

```diff
@@ -0,0 +1,43 @@
+import { Customer, CustomerTypeEnum } from '~/generated/graphql'
+
+export const getInitials = (str: string) =>
+  str.split(' ').reduce((acc, n) => (acc = acc + n[0]), '')
+
+export const computeCustomerInitials = (
+  customer?: Pick<Customer, 'name' | 'firstname' | 'lastname'> | null,
+) => {
+  const { name = '', firstname = '', lastname = '' } = customer ?? {}
+
+  if (name) {
+    return getInitials(name)
+  }
+
+  if (firstname || lastname) {
+    return getInitials(`${firstname} ${lastname}`.trim())
+  }
+
+  return '-'
+}
+
+export const computeCustomerName = (
```

Comment:
IIRC the backend was planning to return us this value in a new customer's attribute called `displayName`
See their dive-in: https://www.notion.so/getlago/BE-Differentiate-companies-vs-individuals-for-Lago-customers-abc18d1e050c48b5b86b1a2a3c172565?pvs=4#7b3983ab3632430da86f91b5a25508a8

Reply:
yes displayName should be available on the graphql

Reply:
Yup, added a new commit. Used `displayName` instead of computing the name on the frontend. I've also kept the function, so if in the future we would need to change anything related to customer names, it would be way easier.